src/pages/MyProfilePage.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import './MyProfilePage.scss';
+
+function MyProfilePage() {
+  const rockets = useSelector((state) => state.rockets);
+  const missions = useSelector((state) => state.missions);
+  const dragonsList = useSelector((state) => state.dragons);
+  const filteredDragons = dragonsList.filter((dragon) => dragon.dragonReserved);
```

Comment:
Please ensure that you render `joined missions` and `reserved rockets` using `filter();` just as you have done with `joined dragon` according to the project requirement.

Reply:
Thanks for the comments. The changes have been applied. ✅